### PR TITLE
Drag and drop

### PR DESCRIPTION
**Pull request by andersonlebon**
- [ ] Add 2 new JavaScript files and import them as modules:
- one will contain methods related to drag / drop and sorting functionality.
- the other will contain methods related to the status updates (completed: true / false).
- [ ] Add event listeners to list items (dragstart, dragover, drop) and to checkbox (change).
- [ ] Update items object's values for index and completed keys upon user actions.
- [ ] Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/javascript/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (carlos-ssh) [inline .github/workflows/linters.yml]: - Kindly enable the [right linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) in your branch.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,62 @@
...
+- HTML;
+- CSS;
+- JavaScript.
+
+## Getting Started
+
+To get the content of this project locally you need to run this command in your terminal:
+
+- `https://github.com/andersonlebon/todo-list.git`
+
```
Comment (carlos-ssh) [inline README.md:26]: - Would be great to have instructions for non-technical people in your README file.
And hoe they can follow steps to install and run your project.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,62 @@
...
+## :handshake: Contributing
+
+Feel free to check the [issues page](https://github.com/andersonlebon/todo-list/issues)
+
+## Show your support
+
+Give a :star: if you like this project!
+
+## Acknowledgments
```
Comment (carlos-ssh) [inline README.md:66]: - Missing a license.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @andersonlebon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (andersonlebon) [reply README.md:66]: Hello @carlos-ssh  thanks for the review, I have done what you said may you please help me check and see if everything is good before I request the second review?

Comment (carlos-ssh) [reply README.md:66]: Nice!

Comment (carlos-ssh) [reply README.md:66]: If you solved the requested changes request another code review in your dashboard.

Comment (andersonlebon) [reply README.md:66]: Thanks sir.

**Review by carlos-ssh — APPROVED** [review]
Hi @andersonlebon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**